rust/rope/src/multiset.rs

```diff
@@ -327,6 +333,54 @@ impl Subset {
             subset_amount_consumed: 0,
         }
     }
+
+
+    /// Print a debug representation where `#` for count 1, `-` for 0, and
+    /// digits otherwise. Mainly useful for testing with small lengths.
+    pub fn debug_print(&self) {
```

Comment:
Ooops, I deleted this but it was reintroduced by a faulty rebase and I didn't notice it.